### ISSUE TITLE
nREPL

### ISSUE THREAD

**Issue by blindcoding9** [djblue/portal#179]
I'm trying to run the REPL middleware but I'm not able to.

Shouldn't I see the result of (range 10) in Portal?

project.clj

```clojure
(defproject clean "0.1.0-SNAPSHOT"
  :description "FIXME: write description"
  :url "http://example.com/FIXME"
  :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
            :url "https://www.eclipse.org/legal/epl-2.0/"}
  :dependencies [[org.clojure/clojure "1.11.1"]
                 [djblue/portal "0.40.0"]
                 [ch.qos.logback/logback-classic "1.4.5"]]
  :repl-options {:nrepl-middleware [portal.nrepl/wrap-portal]})
```

core.ns

```clojure
(ns clean.core
  (:require [portal.api :as p]
            [portal.api :as p]))

(p/open)

(add-tap portal.api/submit)

(tap> 123)
(range 10)
```
portal

![image](https://user-images.githubusercontent.com/84873607/234265896-3b8a4821-f23f-487f-b525-5bb0631db988.png)

**Comment by djblue** [comment]
Since editor tooling will also use eval and it causes a lot of noise being sent to portal. The [nREPL middleware check if a file is included for an eval](https://github.com/djblue/portal/blob/master/src/portal/nrepl.clj#L63) to distinguish between tooling and user evals.

**Comment by djblue** [comment]
I got in a fix for this via https://github.com/djblue/portal/commit/6d2639f7629d2f8ae8bc80e9dd7ad9b2a1fb3e40 which will be part of the next release.

**Comment by djblue** [comment]
Fixed in [`0.58.0`](https://github.com/djblue/portal/releases/tag/0.58.5).
